app/app/settings.py

```diff
@@ -568,10 +568,11 @@
 # Chat
 CHAT_PORT = env('CHAT_PORT', default=8065)  # port of where mattermost is hosted
 CHAT_URL = env('CHAT_URL', default='localhost')  # location of where mattermost is hosted
+CHAT_SERVER_URL = env('CHAT_SERVER_URL', default='chat')  # location of where mattermost is hosted
```

Comment:
dan says

@Owocki @thelostone-mc @octavioamu I'm going to merge @androolloyd’s chat PR into stable - it requires additional ENV configs to run locally: CHAT_URL=localhost and CHAT_SERVER_URL=chat

should we add these to local.env?

Reply:
great idea.